shaolinmkz/fast-food-fast
# #160741253 Have all end points functional

#### What does this PR do?
Have all required endpoints working

#### Description of Task to be completed?
Having the following URL's functioning:
*  `Place order` route 
* `GET all orders` route
* `GET a specific` orders route
* `GET order history` route
* `Update an order status` route
* Write tests to get over 90% coverage
* Secure all routes required to be safe

#### How should this be manually tested?
After cloning the repo, cd into it and run `npm test` to see the test coverage then run `npm start:dev` or `npm start` to start the application.
Use Postman to test all end points above with the` json` header type.

#### Any background context you want to provide?
I am not fully done with the test and clean-ups, but i want to show you my progress.

#### What are the relevant pivotal tracker stories?
[#160587840](https://www.pivotaltracker.com/story/show/160587840)
[#160800110](https://www.pivotaltracker.com/story/show/160800110)
[#160799939](https://www.pivotaltracker.com/story/show/160799939)
[#160800229](https://www.pivotaltracker.com/story/show/160800229)
[#160800337](https://www.pivotaltracker.com/story/show/160800337)
